### PR TITLE
Make variant creator a full page view

### PR DESCRIPTION
**Pull request by dominik-zeglen**
There are enough changes to make it a separate PR for reviewer's comfort. Next PR will introduce new features (stock management).

### Pull Request Checklist

1. [x] All visible strings are translated with proper context.
1. [x] All data-formatting is locale-aware (dates, numbers, and so on).
1. [x] Translated strings are extracted.
1. [x] Number of API calls is optimized.
1. [x] The changes are tested.
1. [x] Type definitions are up to date.
1. [x] Changes are mentioned in the changelog.

### CODE REVIEW

File: src/products/components/ProductVariantCreatorPage/ProductVariantCreatorPage.tsx [hunk 1]
```diff
@@ -69,61 +64,54 @@ function canHitNext(
...
-      case "summary":
-        setStep("prices");
-        break;
-    }
-  }
+const ProductVariantCreatePage: React.FC<ProductVariantCreatePageProps> = props => {
+  const { attributes, defaultPrice, errors, onSubmit, ...contentProps } = props;
+  const classes = useStyles(props);
+  const intl = useIntl();
+  const [step, { next, prev, set: setStep }] = useWizard<
```
Comment (krzysztofwolski) [inline src/products/components/ProductVariantCreatorPage/ProductVariantCreatorPage.tsx:108]: Could we use
nextStep
prevStep
setStep
as standard? next/prev/set is hard to track and could be anything (for example next item in the loop)

Comment (krzysztofwolski) [reply src/products/components/ProductVariantCreatorPage/ProductVariantCreatorPage.tsx:108]: Or if we dont pay per character then maybe `navigateNextStep` would be really clear.

File: src/products/components/ProductVariantCreatorPage/ProductVariantCreatorPrices.tsx [hunk 2]
```diff
@@ -0,0 +1,331 @@
...
+    }
+  }),
+  { name: "ProductVariantCreatorPrices" }
+);
+
+export type PriceOrStock = "price" | "stock";
+export interface ProductVariantCreatorPricesProps {
+  attributes: ProductDetails_product_productType_variantAttributes[];
+  currencySymbol: string;
+  data: ProductVariantCreateFormData;
```
Comment (krzysztofwolski) [inline src/products/components/ProductVariantCreatorPage/ProductVariantCreatorPrices.tsx:44]: I've found name `data` really confusing when juggling few queries hooks and forms same time. `formData` would be more clear to me.

File: src/products/components/ProductVariantCreatorPage/ProductVariantCreatorPrices.tsx [hunk 3]
```diff
@@ -0,0 +1,331 @@
...
+
+  const attributeChoices = attributes.map(attribute => ({
+    label: attribute.name,
+    value: attribute.id
+  }));
+  const priceAttributeValues = data.price.all
+    ? null
+    : data.price.attribute
+    ? attributes
+        .find(attribute => attribute.id === data.price.attribute)
```
Comment (krzysztofwolski) [inline src/products/components/ProductVariantCreatorPage/ProductVariantCreatorPrices.tsx:76]: Data structure is bit funky, could we move this filtering to the utils? `getAttributeValues(attribute)` maybe.

File: src/products/components/ProductVariantCreatorPage/ProductVariantCreatorValues.tsx [hunk 4]
```diff
@@ -0,0 +1,75 @@
...
+                <Debounce
+                  debounceFn={() => onValueClick(attribute.id, value.slug)}
+                  time={100}
+                  key={value.slug}
+                >
+                  {change => (
+                    <ControlledCheckbox
+                      checked={isSelected(
+                        value.slug,
+                        data.attributes.find(
```
Comment (krzysztofwolski) [inline src/products/components/ProductVariantCreatorPage/ProductVariantCreatorValues.tsx:53]: Another candidate for attr getter probably

Comment (dominik-zeglen) [reply src/products/components/ProductVariantCreatorPage/ProductVariantCreatorValues.tsx:53]: Actually this is a call to util func

**Pull request merged**